Sources/Datadog/Versioning.swift

```diff
@@ -0,0 +1,3 @@
+// GENERATED FILE: Do not edit directly
```

Comment:
The CI fails on missing license header in this file:
```
🔥 Missing the license header in files: 
./Sources/Datadog/Versioning.swift
```

Let's exclude this file in [check-license.sh](https://github.com/DataDog/dd-sdk-ios/blob/master/tools/license/check-license.sh) 👍.

Reply:
yep, thanks for the tip